experimental/auth-logic/main.go

```diff
@@ -26,11 +27,13 @@ import (
 // compiler can then run on the generated code.
 func main() {
 
-	appName := os.Args[1]
-	endorsementFilePath := os.Args[2]
-	provenanceFilePath := os.Args[3]
-	outputFilePath := os.Args[4]
-	queryName := os.Args[5]
+	appName := flag.String("app-name", "", "set name of application to be released")
```

Comment:
```suggestion
	appName := flag.String("app_name", "", "name of application to be released")
```

Based on what we have in the rest of the code, but I am not sure what is the more general convention.

Reply:
OK it looks like you are right that underscores should be used according to an internal style guide.